src/App.jsx

```diff
@@ -14,70 +15,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function clearState() {
+    setGood('');
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {!selectedGood
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+        : (
```

Comment:
This syntax looks better in my opinion
```suggestion
      {!selectedGood ? (
          <h1 className="title is-flex is-align-items-center">
            No goods selected
          </h1>
        ) : (
```